### PR TITLE
Misc. link doc updates

### PR DESCRIPTION
**Pull request by mmcev106**

### CODE REVIEW

File: pages.md [hunk 1]
```diff
@@ -6,47 +6,31 @@ Note: When building links to module pages in module code, make sure to use the `
...
+Links to pages can be configured to appear in REDCap's left-hand menu by adding them to `config.json`.  These can include module pages or links to outside websites.  Links configured under the `control-center` section (shown below) will appear only in REDCap's **Control Center**. Links configured under the `project` section (shown below) will be visible by default for users with design rights, on all project pages, on projects where the module is enabled.  Display behavior/permission and link details (name, url, etc.) can be modified on the fly via the `redcap_module_link_check_display()` hook (see the [method documentation](methods/README.md#em-hooks) for details).  See [the config.json docs](config.md) for details on link configuration options.
```
Comment (grezniczek) [inline pages.md:9]: Maybe highlight the restriction to design rights users more:

`... will be visible by default **only for users with design rights**, on all project pages, on projects ...`

**Comment by grezniczek** [comment]
@mmcev106  - Other than that, looks good.

Can you remind me, how contributing to the module docs should work? You mentioned "sub repo", but I have not yet looked into what sub repos mean. I naively thought that the "docs" folder in the EM Framework probaby constitutes a sub repo and any commits versus this folder would end up at the correct place. Is this the case?
Or do I have to issue PRs against https://github.com/vanderbilt-redcap/external-module-framework-docs specifically?

Or is this a "special case" because you wanted this in the merge-UofF-guide branch?

File: pages.md (hunk 1 above)
Comment (mmcev106) [reply pages.md]: Done!

**Comment by grezniczek** [comment]
I see. This is great! Thanks for explaining.

**Pull request merged**